doc/source/serve/llm/user-guides/deployment-initialization.md

```diff
@@ -1,17 +1,20 @@
-(model-loading-guide)=
-# Model loading
+(deployment-initialization-guide)=
+# Deployment Initialization
 
-Configure model loading from Hugging Face, remote storage, or gated repositories.
+The initialization phase of a serve.llm deployment involves many steps, including preparation of model weights, engine (vLLM) initialization, and Ray serve replica autoscaling overheads. A detailed breakdown of the steps involved in using serve.llm with vLLM is provided below.
 
-Ray Serve LLM supports loading models from multiple sources:
+## Startup Breakdown
+- **Node Acquiring**: If a GPU node isn't available, a new instance must be provisioned. 
+- **Fixed Ray/Node Initialization**: Ray/vLLM incurs some fixed overhead when spawning new processes to handle a new replica, which involves importing large libraries (such as vLLM), preparing model and engine configurations, etc.
+- **Memory Profiling**: vLLM runs some inference on the model to determine the amount of available memory it can dedicate to the KV cache
+- **Warmup**: vLLM runs additional inference to warm up the KV cache.
+- **CUDA Graph Capture**: vLLM handles Cuda Graph capture independently of Torch Compile. More details are [here.](https://docs.vllm.ai/en/latest/design/cuda_graphs.html)
+- **Model Loading**: Retrieve model from source to GPU memory, either from Hugging Face or cloud storage. 
```

Comment:
You need to separate downloading the model, from loading the model. They can happen in a fused operation (using runai, etc). Make this clear.